### ISSUE TITLE
Publish WCAG training content to Platform Website

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28775]
## Issue Description
In Sprint 58, @1Copenut hosted 2 training sessions on the new WCAG 2.1 changes. At this time, the WCAG 2.1 training content only exists in slide decks, plus the 2 recordings of the trainings. Per [guidance](https://vfs.atlassian.net/wiki/spaces/SDT/pages/1802993669/Outline+Platform+training+framework+draft) written by the Service Design team, "Training should be documented on the platform website as written content with videos and slide decks provided as supplemental information." We need to create a location on Platform Website to document the information from the slide decks and training into written text, ensure the training recordings are published to YouTube, and communicate to VFS teams where to find this information on Platform Website.

---
## Tasks
- [x] Determine (with @JoeTice) where the WCAG 2.1 content should live in Platform Website
- [x] Create new Platform Website page (if needed)
- [x] Provide written instructional content for WCAG 2.1 on the Platform website.
- [x] Upload the recordings to the Platform Youtube channel (follow [guidelines in this doc](https://vfs.atlassian.net/wiki/spaces/SDT/pages/1802993669/Outline+Platform+training+framework+draft))
- [x] Embed the video recordings into the new Platform Website page

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by shiragoodman** [comment]
@JoeTice my idea would be to create a WCAG space under Accessibility Testing here:
![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/2a46312d-9c36-4900-9058-0e5138a2bfd7)

Let me know what you think.

**Comment by shiragoodman** [comment]
obviously the slide decks page would be deleted and incorporated into the WCAG space.

**Comment by 1Copenut** [comment]
@shiragoodman First page first draft is done as of EOD Friday, August 20.

**Comment by peggygannon** [comment]
Hi, @shiragoodman @1Copenut I've reviewed and edited these two drafts of WCAG 2.1 training docs. Please let me know if you have any questions about my edits/queries. 

https://vfs.atlassian.net/wiki/spaces/PPST/pages/1836253234/Copy+of+WCAG+2.1+new+success+criteria+WIP
https://vfs.atlassian.net/wiki/spaces/PPST/pages/1844281369/Copy+of+WCAG+2.1+Foundational+Testing+Redux

**Comment by 1Copenut** [comment]
Thank you @peggygannon !

**Comment by shiragoodman** [comment]
thank you @peggygannon ! Looks like you only have 1 comment in the success criteria page regarding merging 2 sections, and I agree. 

But before you make and changes @1Copenut, I really think the page for success criteria can be merged with the page for foundational testing. For example, with regards to success criteria 1.3.4, why can't it be written out:

- Learn more link
- Short description
- Key takeaways
- How to test
- Success means

Like this:
![image](https://user-images.githubusercontent.com/71330581/130676951-d7a1512c-2637-4412-ac11-0ef86cb180c8.png)
![image](https://user-images.githubusercontent.com/71330581/130677172-0e594633-a190-45ab-8693-21eec2cf6025.png)
